spec/requests/v1/files_spec.rb

```diff
@@ -116,6 +117,8 @@
       patch "/v1/files/#{file.id}/entries/#{entry.id}", params: params.to_json, headers: headers
 
       expect(response).to have_http_status(:no_content)
+      expect(entry.reload.aasm.current_state).to eql :errored
```

Comment:
`ActsAsStateMachine` adds some query methods to the model, e.g. `errored?`, so this would also work:

`expect(entry.reload).to be_errored`